aws-android-sdk-appsync/src/main/AndroidManifest.xml

```diff
@@ -3,4 +3,9 @@
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
```

Comment:
Should we document that our SDK would require these permissions?

Reply:
Restructured the project. These permissions are not needed. Removed.